cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -1193,6 +1193,44 @@
     </description>
   </property>
 
+  <property>
+    <name>router.connection.backlog</name>
+    <value>20000</value>
+    <description>Determines the connection backlog in the Router Server.</description>
+  </property>
+
+  <property>
+    <name>router.server.boss.threads</name>
+    <value>1</value>
+    <description>Deterermines the number of boss threads in the Router Server.</description>
+  </property>
+
+  <property>
+    <name>router.server.worker.threads</name>
+    <value>10</value>
+    <description>Determines the number of worker threads in the Router Server.</description>
+  </property>
+
+  <property>
+    <name>router.client.boss.threads</name>
+    <value>1</value>
+    <description>Determines the number of boss threads in the Router Client.</description>
+  </property>
+
+  <property>
+    <name>router.client.worker.threads</name>
+    <value>10</value>
+    <description>Determines the number of worker threads in in the Router Client.</description>
+  </property>
+
+  <property>
+    <name>router.connection.timeout.secs</name>
+    <value>15</value>
```

Comment:
I think the original problem that this PR is trying to solve is for inflight request with the firewall blocking the forwarding request from the router to the cluster. This mean timeout also needs to apply for inflight requests.